index.html

```diff
@@ -0,0 +1,124 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./css/style.css">
+
+    <title>Educational Robotic Meeting in Africa</title>
+  </head>
+  <body>
+    <div class="header">
+      <ul class="flex-row">
+        <li><a href="#"><img src="./icons/facebook.png" alt="facebook"></a></li>
+        <li><a href="#"><img src="./icons/twitter.png" alt="twitter"></a></li>
+        <li><a href="#">English</a></li>
+        <li><a href="#">My Page</a></li>
+        <li><a href="#">Logout</a></li>
+      </ul>
+    </div>
+    <nav id="navbar" class="navbar">
+      <div id="overlay" class="overlay"></div>
+        <div class="navbarBox">
+          <a class="navbarBrand" href="index.html"><img src="./images/logo.png" alt="logo"></a>
+          <img src="./icons/hamburger.png" alt="open menu" id="openNavbar" class="openNavbar">
+          <img src="./icons/close.png" alt="close Menu" id="closeNavbar" class="closeMobileMenu closeNavbar" >
+          <div class="navbarListBox flex-row" id="navbarListBox">
+            <ul id="menuList" class="menuList">
+              <li class="navItem">
+                <a class="navLink" aria-current="page" href="about.html">About</a>
+              </li>
+              <li class="navItem">
+                <a class="navLink" href="#">Program</a>
+              </li>
+              <li class="navItem">
+                <a class="navLink" href="#" aria-disabled="true">Join</a>
+              </li>
+              <li class="navItem">
+                <a class="navLink" href="#" aria-disabled="true">Sponsor</a>
+              </li>
+              <li class="navItem">
+                <a class="navLink" href="#" aria-disabled="true">News</a>
+              </li>
+            </ul>
+            <div id="campaign" class="campaign"><a class="campaignLink" href="#"> Campaign</a></div>
+          </div>
+        </div>
+    </nav>
+
+    <section class="headline flex-column">
+        <div class="introductionText">
+            <h1>"Hello! Teaching world"<span>EDUCATIONAL ROBOTIC MEETING IN AFRICA 2023</span></h1>
```

Comment:
- [ ] `"Hello! Teaching world"` is a small line of text and as such shouldn't be part of the main page title. It should be separated for page structure purposes. You could use an `hx` tag

<i>Note That this applies to the about page</i>